### DISCUSSION TITLE
[Feature Request] Ability to create groups or tags in the history sidebar

### DISCUSSION THREAD

**Discussion by peterhoang** [danny-avila/LibreChat#3081] [Feature Requests & Suggestions]
I would greatly appreciate the ability to categorize my chat history using groups or tags, as I utilize AI for a wide range of purposes and tasks, and this feature would help me to better manage and reference my conversations.

**Comment by danny-avila** [comment]
Yes, this has been highly requested, I would like to implement it sooner rather than later.
